tests/testthat/test-ui-scatter3D.R

```diff
@@ -11,6 +11,8 @@ test_that("cXscatter3d2", {
 
 test_that("cXscatter3d3", {
     check_ui_test(cXscatter3d3())
+
+    warning("The loess surface fit appears in an unusual place but it is working as designed")
```

Comment:
Please make this a message instead of a warning, and we need to update to make it clear that when the user transposes the data the loess surface fit is in an unusual location - this makes it sound like the loess surface is odd all the time.

Reply:
ok, have updated the message